packages/beacon-node/src/network/network.ts

```diff
@@ -38,7 +39,7 @@ interface INetworkModules {
 
 export class Network implements INetwork {
   events: INetworkEventBus;
-  reqResp: IReqRespBeaconNode;
```

Comment:
why do we have to switch to implementation instead of interface?

Reply:
Removed unwanted diff